beacon-chain/blockchain/execution_engine.go

```diff
@@ -282,7 +282,9 @@ func (s *Service) getPayloadAttribute(ctx context.Context, st state.BeaconState,
 			logrus.WithFields(logrus.Fields{
 				"validatorIndex": proposerID,
 				"burnAddress":    fieldparams.EthBurnAddressHex,
-			}).Error("Fee recipient not set. Using burn address")
+			}).Warn("Fee recipient not set. Using burn address. " +
+				"You will not be rewarded transaction fees on this setting. " +
+				"Please set a different fee recipient.")
```

Comment:
Let's avoid periods in our logs. Also, we should perhaps tell them to read our documentation regarding fee recipients?